### DISCUSSION TITLE
Whatsapp Notification

### DISCUSSION THREAD

**Discussion by anoop-mathew** [Budibase/budibase#6802] [Ideas]
Similar to discord, if we can have whatsapp also to be added, for sending custom message as part of automation

**Comment by PClmnt** [comment]
Unfortunaterly, unlike Discord, WhatsApp doesn't appear to have an open API. There is WhatsApp for business, but it's unlikely we'd provide an integration for that. 

If you have access to the WhatsApp Business platform, you could use an External Data Connector block in conjunction with the API they provide to suit your needs.
